### ISSUE TITLE
MacOS: HUD settings in Preferences do not do anything

### ISSUE THREAD

**Issue by ArchKevin22** [TASVideos/desmume#43]
In the Preferences menus, the HUD boxes don't seem to do anything. I don't know whether it's because it's reading from another settings file, but the HUD settings only seem to work when changing them from the View settings in the menu bar. Hopefully the pictures illustrate what I'm talking about.

<img width="736" alt="screen shot 2017-03-02 at 10 26 04 am" src="https://cloud.githubusercontent.com/assets/19791612/23521184/c60900d4-ff32-11e6-9ad2-d492971227c4.png">
<img width="585" alt="screen shot 2017-03-02 at 10 26 31 am" src="https://cloud.githubusercontent.com/assets/19791612/23521185/c620d574-ff32-11e6-973f-a76650b26b27.png">

Also, where is the settings file located? As in, when I save the settings, where does it go? I can't seem to find it in ~/Library.

**Comment by rogerman** [comment]
You have a misunderstanding about how the Mac port treats its default settings, so here's the explanation.

In the Mac port, it is possible to have multiple display windows open at once, and so display window settings must be tracked on a per-window basis. The View menu reflects the current settings of the display window that is currently in focus.

Whenever the user creates a new display window via the File > New Display Window menu option, it will automatically use the default settings from DeSmuME Preferences upon the window's creation. These default settings are read from DeSmuME's user defaults file, located at "~/Library/Preferences/org.desmume.DeSmuME.plist".

If the user wants to change the default HUD settings, they can do so by using one of these two methods:
1. Go into DeSmuME Preferences, under Display and HUD Settings, and then modify the default settings there (exactly as shown in your provided screenshot).
2. Create a new display view with File > New Display Window, then go to View > Show HUD Settings. In the HUD Settings panel, choose the desired settings (which will immediately apply to the current window), then click Save Settings as Default. If you already have the DeSmuME Preferences window open, you will notice that clicking Save Settings as Default will immediately update the HUD Settings in DeSmuME Preferences.

In short:
1. The View menu only reflects the display window that currently has focus, and changes made through here will immediately apply to the current display window.
2. The display settings in DeSmuME Preferences are default settings that apply only for new display windows that are created via File > New Display Window.

After retesting all of this, the HUD Preferences work exactly as intended. Closing, since this is a user misunderstanding, and not an actual bug.

**Comment by ArchKevin22** [comment]
Oh okay sorry about that.
